test/TestAsset/Extension/FooTwigExtension.php

```diff
@@ -9,9 +9,11 @@
 
 namespace ZendTest\Expressive\Twig\TestAsset\Extension;
 
-class FooTwigExtension extends \Twig_Extension
+use Twig\Extension\AbstractExtension;
+
+class FooTwigExtension extends AbstractExtension
 {
-    public function getName()
+    public function getName(): string
```

Comment:
Our CS currently has spaces on either side of a `:` when used to denote a typehint.

Reply:
resolved.